### DISCUSSION TITLE
If some of the data to be stored requires encryption how would that work?

### DISCUSSION THREAD

**Discussion by blackduck-joe** [seaweedfs/seaweedfs#2738] [Q&A]
Interested in Seaweed FS as a common abstraction for an unknown underlying file system (not the same customer to customer).  A subset of this data will need encryption.  Is there a built in solution?  How have others addressed this?

**Comment by chrislusf** [comment]
see https://github.com/chrislusf/seaweedfs/wiki/Filer-Data-Encryption
